crates/nu-command/tests/commands/group_by.rs

```diff
@@ -97,9 +97,9 @@ fn errors_if_column_not_found() {
 }
 
 #[test]
-fn errors_if_input_empty() {
-    let actual = nu!("group-by date");
-    assert!(actual.err.contains("expected table from pipeline"));
+fn group_by_on_empty_list_returns_empty_record() {
+    let actual = nu!("[] | group-by a");
+    assert!(actual.err.is_empty());
```

Comment:
maybe the example is not very good, but you could pass the output of your command, here `group-by` to `to nuon` to get a raw string, and then check that in `assert` :+1: 

```suggestion
    let actual = nu!("[{a: "foo", b: 1}, {a: "bar", b: 2}, {a: "foo", c: 3}] | group-by a | to nuon");
    assert_equal!(actual.out, "{foo: [{a: foo, b: 1}, {a: foo, c: 3}], bar: [[a, b]; [bar, 2]]}");
```

you might also want to format this and possibly use the `pipeline` function to break the pipeline on multiple lines :relieved:

Reply:
Good call! Surprised about the recusrion in `nu!` due to maybe the line breaks.

Reply:
happy to help @0scvr :relieved: